Embedded/MaxMix/Config.h

```diff
@@ -16,19 +16,10 @@
 //********************************************************
 // *** DEFINES
 //********************************************************
-#ifndef VERSION_MAJOR
-    #define VERSION_MAJOR 0
+#ifndef VERSION
+    #define VERSION "\"0.0.0\""
```

Comment:
Do we want the string to contain string notation? This will print FW: "0.0.0" on the info screen for example.

Reply:
it doesn't actually, i tested it 🤷